### ISSUE TITLE
Move the chisel slightly

### ISSUE THREAD

**Issue by lawsonw** [pokeclicker/pokeclicker#2283]
**Current Behavior**
Currently, the chisel (after the recent graphical update for it) is lower than it previously was, and it immediately threw me off. Rather than the "click" being in the middle of the chisel like previously. Now, even when the chisel is 90% over a tile, it's actually pointing at the tile below it.

**Improved Behavior**
Move the graphic for the chisel up, or move its hitbox, so the hitbox is in the middle of the chisel rather than the bottom.

**Reason for change**
It's a very noticeable inconvenience that seems like it could be fixed easily enough.

**Comment by Aegyo** [comment]
I was quite literally about to start this :+1:

**Comment by Aegyo** [comment]
Hmm, after looking into it properly, the hitbox seems to be on the "tip" of the tool, the part you would use to strike something if it were real - this seems intentional, and probably makes more sense for new players.

**Comment by mrmayer97** [comment]
mouses normally have their hitbox at the top; going from a cursor at the top while playing the rest of the game to the hitbox being on the bottom while mining is pretty offputting.

**Comment by lawsonw** [comment]
![86BB08E7-6631-404B-A945-7F0465231A00](https://user-images.githubusercontent.com/77951328/170603141-f02cefd2-db20-472b-ad58-132bbcbd1ff3.jpeg)
My suggestion is to put it roughly where the handle meets the chisel part of the tool.

**Comment by Morat48** [comment]
> That's because the pointy bit of the mouse is at the top, no?
> 
> I might be misunderstanding your initial suggestion, because that seems more confusing than the current behavior. Are you able to mark on a picture where you think the hitbox ought to be?

Yes, that is probably why the active area of a typical mouse cursor is at the top left corner. But I think that's probably irrelevant. Convention and muscle memory will lead the player to expect the click to happen at the top left corner of the cursor. Anything else is confusing.

I would move the chisel's hitbox either to the middle as illustrated or to the tip of the little wooden bit sticking out from the top of the tool.

If you aren't convinced, I urge you to just try it out for a bit and see what feels more natural.

**Comment by Melvar** [comment]
When I started playing a few weeks ago, I was very put off by the fact that at the time, the click point of the chisel was at the top left corner of its bounding box. At the time, I was very much expecting that the click point would be at the point of the chisel. Of course, now it’s annoying that it changed, but I’m with Aegyo that for a new player it’s probably better now.

**Comment by everentropy** [comment]
I agree, being able to switch between the two would be more helpful than changing the hitbox for me
